lib/modules/apostrophe-templates/index.js

```diff
@@ -696,5 +696,29 @@ module.exports = {
       req.data.aposBodyClasses = (req.data.aposBodyClasses ? (req.data.aposBodyClasses + ' ') : '') +
         bodyClass;
     };
+
+    // Add a body attribute to be emitted when the page is rendered. This information
+    // is attached to `req.data`, where `req.data.aposBodyDataAttributes` is built up
+    // using `name` as the attribute name which is automatically prepended with "data-"
+    // and the optional `value` argument
+    // Also receives a single object or an array of objects with name and value key pairs
```

Comment:
I think a better use for an object syntax would be an object like:

```
{
  foo: 'value',
  bar: 'other value'
}
```

That feels natural for adding attributes. In fact I think the array syntax could be omitted in favor of this.